src/telemetry/commandTelemetryProps.ts

```diff
@@ -57,3 +53,16 @@ export interface UpdateImageTelemetryProps extends AzdTelemetryProps, ImageSourc
     revisionMode?: KnownActiveRevisionsMode;
     skippedRegistryCredentialUpdate?: 'true';
 }
+
+export interface IngressTelemetryProps {
+    dockerfileExposePortRangeCount?: string;  // IngressPromptStep
+    enableIngress?: 'true' | 'false'; //IngressPromptStep
+    suggestedTargetPort?: string; //getDefaultPort
+    targetPort?: string; //TargetPortInputStep
+}
+
+export interface CreateContainerAppTelemetryProps extends ImageSourceTelemetryProps, EnvironmentVariableTelemetryProps, AzdTelemetryProps, IngressTelemetryProps {
+    // Ingress
+    enableIngress?: 'true' | 'false'; //IngressPromptStep
```

Comment:
Couple things, I just added a PR removing the extra `EnvironmentVariableTelemetryProps` file, so you can just extend `ImageSourceTelemetryProps` now and it will cover the environment variables!

Also, now that you added `IngressTelemetryProps` I think you can probably remove these single ingress properties.

If we end up just extending and not needing to add any more custom properties, it might look cleaner to do something like:

`export type CreateContainerAppTelemetryProps = ImageSourceTelemetryProps & AzdTelemetryProps...`